test/test.js

```diff
@@ -50,6 +51,15 @@ describe("with processors", () => {
         return `
           return ${htmlConstructor}(globalObject, interfaceName);
         `;
+      },
+      processReflect(idl, implObj) {
+        const reflectAttr = idl.extAttrs.find(attr => attr.name === "Reflect");
+        const attrName = reflectAttr.rhs && reflectAttr.rhs.value.replace(/_/g, "-") || idl.name.toLowerCase();
```

Comment:
I see this in action in the output, but don't see the effect of the returned object...

Reply:
See above.